internal/backend/backend.go

```diff
@@ -100,6 +95,16 @@ func (b *Backend) RemoveUser(ctx context.Context, userID string) error {
 
 	delete(b.users, userID)
 
+	if removeFiles {
+		if err := b.storeBuilder.Delete(filepath.Join(b.dir, "store"), userID); err != nil {
+			return err
+		}
+
+		if err := db.DeleteDB(filepath.Join(b.dir, "db"), userID); err != nil {
```

Comment:
Have you considered using a constant for the store and db path so that they remain in sync with other locations?

Reply:
added functions `getStoreDir()` and `getDBDir()` which are used everywhere to get the paths.